pkgs/development/tools/database/liquibase/default.nix

```diff
@@ -72,13 +75,19 @@ stdenv.mkDerivation rec {
       chmod +x $out/bin/liquibase
     '';
 
+  passthru.updateScript = gitUpdater {
+    url = repoUrl;
+    rev-prefix = "v";
+    ignoredVersions = "10.10.10|5.0.0|.*-beta.*";
```

Comment:
Any reason why?

Reply:
I don't actually know where these versions come from. The actual latest releases are in the 4.xx series.  I will add a note, though.

Reply:
Ah, I think these are for the commercial offering.